bedrock/firefox/templates/firefox/interest-dashboard.html

```diff
@@ -88,7 +100,7 @@ <h3 data-accordion-role="tab">{{ _('How is my interest categorized?') }}</h3>
   <div data-accordion-role="tabpanel">
     <p>
       {% trans %}
-        The Interest Dashboard uses both a site-matched and keywords text analysis to categorize interest. The former is based on the categorization of certain URLs we have a high confidence of belonging to a particular interest (i.e. ESPN being Sports). The latter uses statistical analysis of keywords that appear within the title of a webpage mapped to keywords that appear on Wikipedia pages. Those pages are then summarized to IAB’s (Internet Advertising Bureau) content categories. 
+        The Interest Dashboard uses both a site-matched and keywords text analysis to categorize interest. The former is based on the categorization of certain URLs we have a high confidence of belonging to a particular interest (i.e. ESPN being Sports). The latter uses statistical analysis of keywords that appear within the title of a webpage mapped to keywords that appear on Wikipedia pages. Those pages are then summarized to IAB’s (Internet Advertising Bureau) content categories.
```

Comment:
Not sure if this page is localized, but would this white space change get flagged as a string change @flodolo?